### PR TITLE
Fix README, add Synthea seeds, add mock model

### PR DESCRIPTION
**Pull request by katy-sadowski**
This PR contains:

- Some fixes to the README setup instructions, for clarity
- Adds seed files & configuration for a small Synthea database we'll use for testing. Technically dbt tells us not to use seeds for raw data, but I think this is the easiest way for us all to stay in sync with the data we're using to develop the ETL 😄 
- Adds an initial model just for the purpose of testing a dbt run all the way through, to make sure all developers are able to successfully set up the project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,12 +12,20 @@ The project is currently under development and all documentation is aimed at pro
...
+ 2. `cd` into the repo directory and set up a virtual environment:
+ ```bash
+ $ python3 -m venv dbt-env
+ ```
+ - Note that if you're using VS Code, once you set this env as your preferred interpreter for the project, the vscode config in the repo will automatically source the virtualenv each time you open a new terminal in the project.  Otherwise, each time you open a new terminal to use dbt for this project, run:
+```bash
+$ source dbt-env/bin/activate         # activate the environment for Mac and Linux OR
+$ dbt-env\Scripts\activate            # activate the environment for Windows
+```
+ 3. Create a .env file in your repo (`touch .env`) and add a PYTHONPATH entry for your virtual env (for example, if you created your repo in your computer's home directory, the entry will read as: `PYTHONPATH="~/dbt-synthea/dbt-env/bin/python"`)
```
Comment (adambouras) [inline README.md:24]: I think, we can include `source .env` to execute `.env` file in the current shell environment.

File: README.md [hunk 2]
```diff
@@ -38,8 +46,9 @@ synthea_omop_etl:
   target: dev
 ```
 
- 6. Ensure your profile is setup correctly from the command line:
+ 6. `cd` into the project directory, and ensure your profile is setup correctly using dbt debug:
 ```bash
+$ cd synthea_omop_etl
 $ dbt debug
 ```
```
Comment (adambouras) [inline README.md:53]: Should include `dbt deps` to install any required dependencies

File: README.md (hunk 1 above)
Comment (katy-sadowski) [reply README.md]: that's not needed, this is just for VS Code to locate the virtual env. i clarified this in the instructions!

File: README.md (hunk 2 above)
Comment (katy-sadowski) [reply README.md:53]: good catch, thank you!!!!

**Review by adambouras — APPROVED** [review]
I created a PR to address some of issues I found while running the code

**Pull request merged**